esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/app/configure/bin/configure.py

```diff
@@ -13,19 +13,26 @@
         category=UserWarning,
         module="esmvalcore.experimental._warnings",
     )
-    import esmvalcore.experimental as esmvaltool
+    from esmvalcore.experimental.config._config_object import Config
 import yaml
 
-USER_CONFIG_PATH = os.environ["USER_CONFIG_PATH"]
+ESMVALTOOL_DIR = os.environ["ESMVALTOOL_DIR"]
+LATEST_USER_CONFIG_FILE = os.path.join(ESMVALTOOL_DIR,
+                                       "config-user-example.yml")
+USER_CONFIG_OUTPUT_PATH = os.environ["USER_CONFIG_PATH"]
 
 
 def main():
     """Write the updated configuration values to the file defined by
     ``USER_CONFIG_PATH``."""
-    # Get the default configuration values from ESMValTool. A dictionary is
+    # Get the latest configuration values from ESMValTool. A dictionary is
     # needed here to avoid the config object from converting paths to PosixPath
     # objects, which causes issues when writing the YAML file.
-    config_values = dict(esmvaltool.CFG)
+    # The dictionary is then updated with the path to the latest checked out
+    # version of config-user-example.yml from the ESMValTool directory.
```

Comment:
Would it be possible to incorporate this comment into the comment above, please?

Reply:
This has been tricky - best effort so far in 2133c52b0c43a28de54e0329d9bb627b0966be17

Reply:
Actually this is better I think: ba0222fcecb1c76879f4c1bdbabfe30a28d098ab